src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -305,6 +305,7 @@ private void RegisterEventHandlers()
         /// Handle requests to reset colors on geometry graphics objects given an id.
         /// </summary>
         /// <param name="objId">id of the object</param>
+        [Obsolete("This will be moved to a new project in a future version of Dynamo")]
         protected virtual void AttachedProperties_RequestResetColorsForDynamoGeometryModel(string objId)
```

Comment:
Why does this need to be moved? It is empty and the overridden method in `HelixWatch3DViewModel` will be moved anyway.

Reply:
Actually, it will not be moved, but it will cease to exist. I copied that message too much :) Good catch!

Reply:
No it's ok. This will be moved. The plan is to, instead of implementing a handler at the `DefaultWatch3DViewModel` level, to provide a virtual method were extending implementations can register their own handlers. This decouples the base class from an event it doesn't handle and provides more flexibility to inheritors.

Reply:
@aparajit-pratap I'd advice you to take a look at the internal wiki documenting the future refactor as well, as the reasons for these obsoletions are better explained there.